plugin.xml

```diff
@@ -2,9 +2,9 @@
 <plugin xmlns="http://www.phonegap.com/ns/plugins/1.0"
     xmlns:android="http://schemas.android.com/apk/res/android"
     id="cordova-sqlite-storage"
-    version="3.2.0-OS3">
+    version="6.0.0-OS1">
```

Comment:
According to the new specification this should not use the `-OSX` suffix but instead use `+<semver>` suffix. Can be fixed here or in a posterior raise commit (which will be tagged).

Reply:
Thanks for bringing this up, I was actually going to raise these versions on a raise commit but this was an oversight, I missed it. I wanted this to have the original, upstream, version to make it explicit from where we're basing our version